### PR TITLE
Added hubspot deal's description.

### PR DESCRIPTION
**Pull request by hasnain-naveed**

### CODE REVIEW

File: ecommerce/core/management/commands/sync_hubspot.py [hunk 1]
```diff
@@ -72,6 +73,11 @@
                 'dataType': 'DATETIME',
                 'targetHubspotProperty': 'closedate'
             },
+            {
```
Comment (ziafazal) [inline ecommerce/core/management/commands/sync_hubspot.py:76]: Do we really need to define `description` property here?

**Review by ziafazal — APPROVED** [review]
👍

File: ecommerce/core/management/commands/sync_hubspot.py (hunk 1 above)
Comment (hasnain-naveed) [reply ecommerce/core/management/commands/sync_hubspot.py:76]: yes, we need to define this property in settings. I have tested it.

**Review by georgebabey — COMMENTED** [review]
Code looks good but I'm wondering if we need to remove LineItems and Products from the `HUBSPOT_ECOMMERCE_SETTINGS`? I worry the call may fail if we try posting and the customer does not have access to these objects in Hubspot.

**Pull request merged**